docker-compose.yml

```diff
@@ -0,0 +1,34 @@
+services:
+  app:
+    image: atharva0418/springboot-gradle-app
```

Comment:
how is this image built? 
when I do docker compose up, it throws the error
"pull access denied for atharva0418/springboot-gradle-app, repository does not exist or may require 'docker login': denied: requested access to the resource is denied"

Can you list down steps on how you are running it locally? At the end, the goal is to run the app using `docker compose up` command.

Reply:
It is built by the springboot feature 'bootBuildImage'(Check build.gradle). Spring takes care of all the dependencies and configures and builds an image of the application. Run 'gradle bootBuildImage' command to build the image.

Reply:
I see, okay. Wasn't aware of this spring boot feature `bootBuildImage`. Great 👍 

I just renamed the imageName and the container name making it more generalized and one does not have to use dockerhub username. 

You can update the readme and add the section on how to run the app via docker.